src/components/Table/components/TableBody/index.js

```diff
@@ -91,6 +93,11 @@ TableBody.propTypes = {
    * Handled internally
    */
   sortHover: PropTypes.number,
+  /**
+   * [PRIVATE] Adds condensed styles to the table.
```

Comment:
I'd like to stick to the [JSDoc](https://devdocs.io/jsdoc/tags-private) conventions here :) 

```suggestion
   * @private Adds condensed styles to the table.
```